components/Excalidraw.tsx

```diff
@@ -0,0 +1,19 @@
+"use client"
+
+import Image from "next/image"
+
+interface Props {
+  text: {
+    raw: string
+    html: string
+  }
+}
+
+export function Excalidraw({ text }: Props) {
+  const canvas = text.raw
+  return canvas ? (
+    <Image src={canvas} width={500} height={500} alt='Coursition - Excalidraw' className='p-6' />
+  ) : (
+    <div dangerouslySetInnerHTML={{ __html: text.html }} />
```

Comment:
**question:** what is the case for raw HTML? When does Excalidraw emit html?

Reply:
Okay, so we just wanted a template when they don't have any diagram yet.